tests/integration/scoring/test_scoring.py

```diff
@@ -10,6 +10,18 @@
 from ..datasetio.test_datasetio import register_dataset
 
 
+@pytest.fixture
+def rag_dataset_for_test(llama_stack_client):
+    register_dataset(llama_stack_client, for_rag=True)
```

Comment:
same thing here -- don't you need to pass `dataset_id` here also?

Reply:
done. dataset_id="test_dataset" by default, so it is not actually needed to pass tests.